tactic/tidy.lean

```diff
@@ -17,20 +17,19 @@ meta def tidy_attribute : user_attribute := {
 
 run_cmd attribute.register ``tidy_attribute
 
-meta def name_to_tactic (n : name) : tactic (tactic string) := 
+meta def name_to_tactic (n : name) : tactic string :=
```

Comment:
The "staging" effect of `tactic (tactic string)` was not necessary, because the result of `name_to_tactic` is not saved anywhere; and actually the old version did slightly more work when one of the tactics succeeds, because it went and looked up all the declarations first, before trying any of the tactics (granted, this is probably very cheap).